cluster/kubevirtci.sh

```diff
@@ -15,22 +15,39 @@
 export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.17'}
 
 KUBEVIRTCI_VERSION='0b941ea5dc647d3aea6f6a6fff95563d6ce0445e'
-KUBEVIRTCI_PATH="${PWD}/_kubevirtci"
+KUBEVIRTCI_REPO='https://github.com/kubevirt/kubevirtci.git'
+KUBEVIRTCI_PATH=${KUBEVIRTCI_PATH:-"${PWD}/_kubevirtci/cluster-up/"}
+KUBEVIRTCI_PARENT_PATH="$(dirname "$KUBEVIRTCI_PATH")"
```

Comment:
It is weird that you setup full path on the previous line and then shorten it here. I'd expect first to setup the parent and then just extend it with the subdirectory

Reply:
I know what you mean,  let me explain my reasoning:
_kubevirtci is using the path with `cluster-up/`, like so:
- _kubevirtci's `KUBEVIRTCI_PATH`: `/tmp/cnao/cluster-network-addons-operator/_kubevirtci/cluster-up/`
- cluster's `KUBEVIRTCI_PATH`: `/tmp/cnao/cluster-network-addons-operator/_kubevirtci/`

it would be more reasonable to change `KUBEVIRTCI_PATH` to the one `cluster` is using, but this would mean doing this change in kubevirtci, which would potentially break every repo using it. 

So I ended up thinking it's not worth it to change it there and I should just make it here like so.
What do you think? should I fix it in kubevirtci?

Reply:
Can we rename this to CLUSTER_PATH then?

Reply:
I think `CLUSTER_KUBEVIRTCI_PATH` would be better, as it points to the _kubevirtci where the cluster is spun. tell me what you think

Reply:
wait.. the vendored already iuse `KUBEVIRTCI_CLUSTER_PATH` .. 
they took all the d=good names :) 

let's stick to `CLUSTER_PATH` then